src/components/editor/Bindings/Row/ErrorIndicator.tsx

```diff
@@ -22,14 +23,28 @@ function BindingsSelectorErrorIndicator({ bindingUUID }: Props) {
         'errors'
     );
 
-    if (bindingErrors?.length > 0 || configErrors?.length > 0) {
+    const sourceBackfillRecommended =
+        useBinding_resourceConfigOfMetaBindingProperty(
+            bindingUUID,
+            'sourceBackfillRecommended'
+        );
+
+    if (
+        bindingErrors?.length > 0 ||
+        configErrors?.length > 0 ||
+        Boolean(sourceBackfillRecommended)
+    ) {
         return (
             <Typography>
                 <WarningCircle
                     style={{
                         marginRight: 4,
                         fontSize: 12,
-                        color: theme.palette.error.main,
+                        color:
+                            bindingErrors?.length > 0 ||
+                            configErrors?.length > 0
```

Comment:
I think we could just check `!Boolean(sourceBackfillRecommended)` here

Reply:
You can. That is what was there initially. I incorrectly assumed you would be opposed to that since you tend to prefer the evaluation of more severe error conditions.